main.go

```diff
@@ -45,11 +46,14 @@ func main() {
 	}
 	log.SetLevel(logLevel)
 
-	log.Info(*args.profitbricksUsername)
-	log.Info(*args.metadataPath)
-	log.Info(*args.mountPath)
-	log.Info(*args.unixSocketGroup)
-	log.Info(*args.version)
+	log.Infof("initialization parameters: "+
```

Comment:
@cgroschupp Could you please use a raw string literal **`** instead of + .
The code and output will be much cleaner and nicely formatted.

Reply:
But I don't want any new lines in the log file.

Reply:
Multiline logs are not very easy to parse with systems like elk/splunk/graylog.

Alternatively, I can create a log line for each parameter.

Reply:
Strings in golang are immutable. Since you are already using `Infof` then please use some other way to concatenate strings.  For example:
```
log.Infof("initialization parameters: %s, %s")
```
We have to think of code optimization as well as of log output.

Reply:
As suggested in previous post.